src/cool_seq_tool/handlers/seqrepo_access.py

```diff
@@ -145,12 +145,16 @@ def chromosome_to_acs(self, chromosome: str) -> tuple[list[str] | None, str | No
         acs = []
         for assembly in ["GRCh38", "GRCh37"]:
```

Comment:
Can do this in a separate PR if you want (will have to import Assembly)
```suggestion
        for assembly in Assembly.__members__:
```

Reply:
actually, this should be a separate issue, `__members__` gives the keys (`"GRCH38"`) not the values (`"GRCh38"`) and that makes some tests fail

Reply:
`[v.value for v in Assembly]` but w/e I'm too lazy at this point

Reply:
Haha I can make a new PR for it